src/lib/CheckoutComponent.ts

```diff
@@ -0,0 +1,87 @@
+import { PAPER_APP_URL } from '../constants/settings';
```

Comment:
Can we call this file CheckoutLink? CheckoutComponent makes it sound like it's for CheckoutElement

Reply:
Do you think just naming it `renderPaperCheckoutLink` is a bad idea? `CheckoutLink` feels like it's suggesting something else as well 🤔